test/inductor/test_torchinductor.py

```diff
@@ -604,6 +604,70 @@ def fn(x, y, z):
 
         self.common(fn, (x, y, z))
 
+    # lowering operation tests
+    def test_add_uint8_tensor(self):
+        def fn(x):
+            return (x + x).to(torch.int16)
+
+        x = torch.tensor([128], dtype=torch.uint8)
+
+        self.common(fn, x)
+
+    def test_add_int8_tensor(self):
+        def fn(x):
+            return (x + x).to(torch.int16)
+
+        x = torch.tensor([120], dtype=torch.int8)
+
+        self.common(fn, x)
+
+    def test_add_int16_tensor(self):
+        def fn(x):
+            return (x + x).to(torch.int32)
+
+        x = torch.tensor([35000], dtype=torch.int16)
```

Comment:
The error is right, `32767` is the maximum possible `int16` value. This error is coming from eager.

Reply:
Thanks - another mistake corrected.